src/java/picard/cmdline/CommandLineParser.java

```diff
@@ -340,6 +340,16 @@ public void usage(final PrintStream stream, final boolean printCommon) {
         }
     }
 
+    private String convertFromHtml(final String textToConvert) {
+
+        String regex1 = "<br />|<p>|</p>|<table>|</table >|<a href='(.*)'>(.*)</a>|<h4>|<pre>|</pre>|<ul>|</ul>|</li>|</tr>|<hr />";
+        String regex2 = "</h4>|<tbody>|</tbody>|<tr>|<th>";
```

Comment:
For the case that an html tag was missed should we replace all remaining "<.*?>" with "" ?